src/Compilers/CSharp/Test/Semantic/Semantics/AnonymousFunctionTests.cs

```diff
@@ -1241,7 +1241,7 @@ void local(Func<nint, nint> fn)
 {
     Console.WriteLine(fn(0));
 }";
-            VerifyInPreview(source, expectedOutput: "1", metadataName: "$Program.<>c.<$Main>b__0_0", expectedIL: @"
+            VerifyInPreview(source, expectedOutput: "1", metadataName: WellKnownMemberNames.TopLevelStatementsEntryPointTypeName + ".<>c.<" + WellKnownMemberNames.TopLevelStatementsEntryPointMethodName + ">b__0_0", expectedIL: @"
```

Comment:
>WellKnownMemberNames [](start = 71, length = 20)

I think it'd be better if tests did not use these constants. Since the symbol names are effectively public API we would want tests to fail if the names change.

Reply:
The names are captured as values for public APIs in the public API files. However, I agree, extra checks are good to have. I will  add one test per name in one of the future PRs.

Reply:
>  I will add one test per name in one of the future PRs.

Actually, I am going to do this in this PR.